easybuild/easyconfigs/m/MPICH/MPICH-3.0.4-GCC-4.8.1-slurm.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'MPICH'
+version = '3.0.4'
+versionsuffix = '-slurm'
+
+homepage = 'http://www.mpich.org/'
+description = """MPICH v3.x is an open source high-performance MPI 3.0 implementation.
+It does not support InfiniBand (use MVAPICH2 with InfiniBand devices)."""
+
+toolchain = {'name': 'GCC', 'version': '4.8.1'}
+
+sources = ['mpich-%(version)s.tar.gz']
+source_urls = ['http://www.mpich.org/static/tarballs/%(version)s']
+
+# MPICH configure wants F90/F90FLAGS to be renamed to FC/FCFLAGS.
+preconfigopts = 'export FC="$F90"; export FCFLAGS="$F90FLAGS"; unset F90; unset F90FLAGS; '
+
+# Build shared libraries
+configopts = '--enable-shared --with-slurm=/usr/lib64/slurm --with-pmi=pmi2 '
```

Comment:
Either way, it's understandable. I'll leave it to you to make the policy decision.